packages/usuario/src/drivers/db/migrations/20220718180952-create-usuarios-table.js

```diff
@@ -0,0 +1,11 @@
+const { UsuarioSchema, USUARIO_TABLE } = require('../models/usuario');
+
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.createTable(USUARIO_TABLE, UsuarioSchema);
+  },
+
+  async down(queryInterface) {
+    await queryInterface.dropTable(USUARIO_TABLE);
+  },
+};
```

Comment:
Are we exporting two functions? if that's the case, let's give them a name.
Could we add a new line here? You can read why it is needed [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)